pkg/mount/mount.go

```diff
@@ -144,9 +144,7 @@ func RecursiveUnmount(target string) error {
 		err = unmount(m.Mountpoint, mntDetach)
 		if err != nil {
 			if i == len(mounts)-1 { // last mount
-				if mounted, e := Mounted(m.Mountpoint); e != nil || mounted {
-					return err
-				}
+				return err
```

Comment:
should we wrap this one as well, or does the produced error message contain enough information?

```suggestion
				return errors.Wrapf(err, "failed to unmount submount %s", m.Mountpoint)
```

Reply:
It does give details about the error.

Reply:
> should we wrap this one

@thaJeztah fortunately, since #38068 we don't have to :man_dancing:

Reply:
Ah, cool